eslint.config.mjs

```diff
@@ -19,6 +19,7 @@ export default tseslint.config(
       "npm/qsharp/src/*.generated.ts",
       "jupyterlab/lib/",
       "jupyterlab/qsharp-jupyterlab/labextension/",
+      "**/.*/",
```

Comment:
this is _probably_ fine, but would it be better to target specifically `.vscode-test-web`?

Reply:
We have other 'hidden' folders that we want to ensure get filtered. Will revisit if we find a counter example.